specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,54 @@
+# SKU Price Details
```

Comment:
Just to document my position here, I would rather call this "SKU Details" to simplify our overall concepts. I understand it's about the lower level SkuPriceId, but that also adds more complexity to the dataset by adding a third level. I would rather see all SKU properties together in a single bucket without introducing a third level (SKU > properties vs. SKU > Price > properties).

Reply:
I support this. In the current exports using FOCUS, SkuId and SkuPriceId from Azure are the same. For Google, SkuPriceId is concatenated with additional data to make a new Id that is not the same as SkuId and is blank where SkuId relates to Tax and adjustments or corrections. So if Providers dont have a different Id for each column then SkuId column should be used and SkuDetails would be better to align to SkuId.

Reply:
Side note: I believe the SkuId == SkuPriceId in Microsoft data is a bug. I need to investigate. Let's not make any decisions based on that. They should be different.

Reply:
Ok will wait on your feedback

Reply:
We previously had 2 columns, SkuDetails and SkuPriceDetails, but after a suggestion and subsequent conversation in TF2, we consolidated them to 1 column. The thought on it being SkuPriceDetails is that if we only have one field, this is more granular and it could hold properties of both SkuId as well as SkuPriceId. SkuDetails wouldn't support any properties of SkuPriceId (e.g. details about tiering start and end values) because that property wouldn't be the same across all SkuPriceIds within that SkuId.

I'm intending a follow-up change to SkuPriceId to indicate that because a number of fields are properties of SkuPriceId (including this and list price, for example), the value for SkuId can be repeated for SkuPriceId, assuming that it doesn't violate any of the corresponding column requirements (one price, one set of properties, etc.)

Reply:
Follow up PR #530 submitted, per above comment